client/src/components/CourseCard.tsx

```diff
@@ -32,9 +36,9 @@ export const CourseCard = ({ course, className, query }: CourseCardProps) => {
         <CourseTerms course={course} variant='small' query={query} />
         <div className='mt-2 text-gray-600 dark:text-gray-400'>
           {query ? (
-            <Highlight text={course.description} query={query} />
+            <Highlight text={courseDescriptionShortened} query={query} />
```

Comment:
I don't think we should shorten the description if a query is present

Reply:
But wouldn't that only shorten descriptions on the native explore page without search then, any time that the user searches, the description remains huge

Reply:
That's ok, I can leave it out